assets/constants.ts

```diff
@@ -6977,6 +6993,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "gsoc_M1.2",
+  "SHORT_COMMIT_HASH": "e526997625"
```

Comment:
Revert this.

Reply:
Actually I didn't made this change. Tho I have reverted it, I am not sure how it was changed.